drivers/v4l-uvc/README.md

```diff
@@ -0,0 +1,44 @@
+# Video4Linux USB Video Class extension
+
+## Installation
+
+See [Installing Extensions](https://github.com/siderolabs/extensions#installing-extensions).
+
+## Usage
+
+Enable the `uvcvideo` module in Talos machine config to enable `/dev/video` devices from USB devices supporting the USB video device class.
+
+```yaml
+machine:
+  kernel:
+    modules:
+      - name: uvcvideo
+```
+
+## Verifiying
+
+You can verify the modules are enabled by reading the `/proc/modules` where it _should_ show the module is live.
+
+For example:
+
+```
+❯ talosctl -n 192.168.42.15  read /proc/modules
+TODO
+```
+
+In addition, you should be able to verify presence of the video device if the USB device is plugged in, checking `/dev` directory.
+
+For example:
+
+```
+❯ talosctl -n 192.168.42.15 ls /dev | grep video
+TODO
```

Comment:
TODO

Reply:
Resolved in https://github.com/siderolabs/extensions/pull/296/commits/5ac513c2387c7ba4089f80a42b89218759fad0ce